packages/opentelemetry-instrumentation-mcp/opentelemetry/instrumentation/mcp/fastmcp_instrumentation.py

```
@@ -30,12 +31,35 @@ def instrument(self, tracer: Tracer):
+        )
+
     def uninstrument(self):
         """Remove FastMCP-specific instrumentation."""
         # Note: wrapt doesn't provide a clean way to unwrap post-import hooks
         # This is a limitation we'll need to document
         pass
 
+    def _fastmcp_init_wrapper(self):
+        """Create wrapper for FastMCP initialization to capture server name."""
+        @dont_throw
+        def traced_method(wrapped, instance, args, kwargs):
```

Ruff (0.13.3)

Unused function argument: `instance`

(ARG001)